event/description.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) MongoDB, Inc. 2024-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package event
+
+import (
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/address"
+	"go.mongodb.org/mongo-driver/tag"
+)
+
+// ServerDescription contains information about a node in a cluster. This is
+// created from hello command responses. If the value of the Kind field is
+// LoadBalancer, only the Addr and Kind fields will be set. All other fields
+// will be set to the zero value of the field's type.
+type ServerDescription struct {
```

Comment:
The ServerDescription holds data from the handshake, parsed from `driverutil.NewDescriptionServer`.